src/vaadin-dialog-draggable-mixin.html

```diff
@@ -19,6 +19,8 @@
         this._stopDrag = this._stopDrag.bind(this);
         this.$.overlay.$.overlay.addEventListener('mousedown', this._startDrag);
         this.$.overlay.$.overlay.addEventListener('touchstart', this._startDrag);
+
+        this.__isMobile = /iPad|iPhone|iPod|Android|webOS|BlackBerry|Windows Phone/i.test(navigator.userAgent);
```

Comment:
Could align this with: https://github.com/vaadin/vaadin-grid/blob/master/src/vaadin-grid.html#L73-L80

Reply:
🤯

Reply:
Good point, overlooked it and found only `_android` and `_ios` 😅
Updated now.

Reply:
Motivation: Moving feature/device detection to a central location is part of the Lit conversion. It will be easier if there are fewer variants around.